agent/config/config_unix.go

```diff
@@ -135,3 +135,7 @@ func (cfg *Config) platformString() string {
 func getConfigFileName() (string, error) {
 	return utils.DefaultIfBlank(os.Getenv("ECS_AGENT_CONFIG_FILE_PATH"), defaultConfigFileName), nil
 }
+
+func isGMSACapable() bool {
```

Comment:
I dont think this is needed.

also `parseGMSACapability()` is checking this env var and setting default to true whereas this is setting to false.
can we remove this?